content/sales.rst

```diff
@@ -0,0 +1,196 @@
+Sales and Marketing
+###################
+
+:menu_order: 008
+
+.. contents::
+
+
+Introduction
+============
+
+This page briefly describe operational Sales details.
+
+The main contact is Tim Adams, the main email group is at sales@proatria.com.
+
+Sales:
+
+* Provides the main software information to prospective customers, forwards
+  additional technical support queries to Support.
+* Introductory email for the documentation, installation guides and downloads
```

Comment:
It's my attempt at writing an intro list about what is involved in Sales.
But if it's not working I can take it out.